styleguide/components/31-molecules/display-switcher/README.md

```diff
@@ -2,23 +2,35 @@
 
 ## When to use this component
 
-Use the display switcher component to let users switch between display modes of one section of content.
+Use the display switcher component when **a page reload** is triggered to let users:
```

Comment:
CC @delrueba 

Apologies in advance for bringing this up again... 

I read the comments in #592 about this and would like to point out that this component doesn't have the same functional purpose as the tabs component: 
* **Tabs**: Show most important content on page load and allow the user to gather more information by switching tabs. Each tab clearly contains its own content. 
* **Display switcher**: Show the content in another way and let the user decide which format he prefers. For example grid vs. list

I would state the the user DOES know what to expect, when he clicks "View results on map", which is: View the current page content on a map.

There's plenty of example of display switchers. For example:
-  [bol.com](https://www.bol.com/nl/l/muziek/N/3132/filter_N/31266+8342+8082+8079/?view=list): no page reload
- [Ebay](https://www.benl.ebay.be/sch/i.html?_from=R40&_sacat=0&_dmd=2&_nkw=nintendo+console&rt=nc&LH_PrefLoc=3): with a page reload

### TLDR;
doesn't matter if a page reload is triggered or not imo..